README.md

```diff
@@ -140,6 +140,9 @@ For apu1 `flashrom` command line looks like that:
 flashrom -p internal -w apu1_x.y.z.rom -c MX25L1605A/MX25L1606E/MX25L1608E
 ```
 
+A full power cycle is required after flashing. See [firmware_flashing.md](docs/firmware_flashing.md)
```

Comment:
@krystian-hebel please make the link to point to the exact `## coreboot.rom flashing` section